scss/_mixins.scss

```diff
@@ -15,6 +15,36 @@
 	@return $aggregated-selector;
 }
 
+/// Sets SVG background retrieved from the image service
+@mixin oFtIconsGetSvg($icon-name, $color: null, $size: 20, $icon-only: false) {
+	@if ($icon-only == false) {
+		display: inline-block;
+		width: $size + 0px;
+		height: $size + 0px;
+	}
+
+	$service-url: 'https://next-geebee.ft.com/image/v1/images/raw/';
+	$icon-url: 'http%3A%2F%2Fnext-geebee.ft.com%2Fassets%2Ficons%2F#{$icon-name}.svg';
+	$query: '?source=ofticons';
+
+	@if $color != null {
```

Comment:
well, when you set that color, you're supposed to use o-colors at that point, which is good because you can use both palette colors and use cases. The other way would only accept palette colors, so I'm not sure